README.md

```diff
@@ -32,19 +32,19 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `npsfile.js` and use `nps` in a single `package.json` script:
 
 **package.json**
 
 ```json
 {
   "scripts": {
-    "start": "package-scripts"
+    "start": "npm-package-scripts"
   }
 }
 ```
 
-**package-scripts.js**
+**npsfile.js**
```

Comment:
Sounds good to me! I'll continue to use the `nps` alias. I'll also make another pull request to change the default config filename. This will be a breaking change so it'll be a major version bump. Anything else major you think should change?